Makefile

```diff
@@ -56,3 +57,13 @@ clean:
 	rm -rf .pytest_cache
 	find . -type d -name __pycache__ -exec rm -rf {} +
 	find . -type d -name .mypy_cache -exec rm -rf {} +
+
+install:
+	yarn install
+	poetry install
+ifeq ($(ARCH),arm64)
+	$(eval ATOMICS := $(shell poetry export | egrep 'atomics==' | awk '{print $$1}'))
+	$(eval VENV := $(shell poetry run poetry env info -p))
+	poetry run pip uninstall atomics -y
+	poetry run pip install $(ATOMICS) --platform=universal2 --no-deps --target $(VENV)/lib/python*/site-packages
```

Comment:
Is `--target` really needed? I wonder since you run pip via poetry, which is going to operate on the virtualenv.

Reply:
>  yes it is required. this `--target $(VENV)/lib/python*/site-packages` hassle is there only to meet the requirement.

I'm curious, where does pip install the package if you omit the `--target` option?

Reply:
it won't install it at all.

```
ERROR: Can not use any platform or abi specific options unless installing via '--target'
```

Reply:
:woozy_face: